docs/csharp/programming-guide/concepts/async/walkthrough-accessing-the-web-by-using-async-and-await.md

```diff
@@ -1,379 +1,371 @@
+    - On the **Text** tab, specify a monospaced font, such as `Consolas` for the **FontFamily**.
+1. Position the text box and the button so that both appear in the **MainWindow** window.
      For more information about the WPF XAML Designer, see [Creating a UI by using XAML Designer](/visualstudio/xaml-tools/creating-a-ui-by-using-xaml-designer-in-visual-studio).
 
-## Add a reference
-
-1. In **Solution Explorer**, highlight your project's name.
-
-2. On the menu bar, choose **Project** > **Add Reference**.
-
-     The **Reference Manager** dialog box appears.
-
-3. At the top of the dialog box, verify that your project is targeting the .NET Framework 4.5 or higher.
-
-4. In the **Assemblies** category, choose **Framework** if it isn't already chosen.
-
-5. In the list of names, select the **System.Net.Http** check box.
+The resulting XAML should resemble the following:
+
+```xaml
+<Window x:Class="AsyncFirstExample.MainWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        Title="Async Example" Height="400" Width="600" MinHeight="300" MinWidth="500">
+    <Grid>
+        <Button x:Name="_startButton" Content="Start" HorizontalAlignment="Center"
+                Margin="0,10,0,0" VerticalAlignment="Top" Width="75" Height="24" />
+        <TextBox x:Name="_resultsTextBox" TextWrapping="Wrap"
+                 FontFamily="Consolas" VerticalScrollBarVisibility="Visible"
+                 Margin="0,45,0,0" />
+    </Grid>
+</Window>
+```
 
-6. Choose the **OK** button to close the dialog box.
+> [!NOTE]
+> Your `x:Class` name may be different, but should match your corresponding *MainWindow.xaml.cs* file class name.
 
 ## Add necessary using directives
 
-1. In **Solution Explorer**, open the shortcut menu for MainWindow.xaml.cs, and then choose **View Code**.
-
-2. Add the following `using` directives at the top of the code file if they're not already present.
+1. In the **Solution Explorer**, open the shortcut menu for *MainWindow.xaml.cs*, and then choose **View Code**.
+1. Replace all of the `using` directives at the top of the code file with the following:
 
     ```csharp
-    using System.Net.Http;
-    using System.Net;
+    using System.Collections.Generic;
+    using System.Diagnostics;
     using System.IO;
+    using System.Linq;
+    using System.Net;
+    using System.Threading.Tasks;
+    using System.Windows;
     ```
 
 ## Create a synchronous app
 
-1. In the design window, MainWindow.xaml, double-click the **Start** button to create the `startButton_Click` event handler in MainWindow.xaml.cs.
-
-2. In MainWindow.xaml.cs, copy the following code into the body of `startButton_Click`:
+1. In the design window, *MainWindow.xaml*, double-click the **Start** button to create the click event handler in *MainWindow.xaml.cs*.
+1. Select the <kbd>F2</kbd> key to rename the handler to `OnStartButtonClick`.
```

Comment:
Based on [this Visual Studio doc](https://docs.microsoft.com/en-us/visualstudio/ide/reference/rename?view=vs-2019), the rename shortcut is CTRL+R, CTRL+R

Reply:
Have you tried <kbd>F2</kbd>? That works in both Visual Studio and VS Code, not sure about <kbd>Ctrl+R</kbd>?

Reply:
I'm not sure, but I think <kbd>F2</kbd> is a ReSharper thing. Is there a way to open Visual Studio with the default settings but without also losing them later?